app/store/services/activities.js

```diff
@@ -95,36 +95,76 @@ export const getAthleteActivities = token => {
   );
 };
 
-export const getRankings = (activities = {}) => {
+export const getRankings = (members = [], activities = {}) => {
```

Comment:
You are probably right! But the value is not there. I do not think that we spent too much time refactoring this part. We should spend most of our time on reloaddux implementation.